creator-kit-3.config

```diff
@@ -0,0 +1,15 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_cc2520=y
+CONFIG_PACKAGE_relay-gateway=y
+CONFIG_PACKAGE_webscripts=y
+CONFIG_IMAGEOPT=y
+CONFIG_VERSIONOPT=y
+CONFIG_VERSION_DIST="OpenWrt"
+CONFIG_VERSION_NICK="Ci40Kit1Project3"
+CONFIG_VERSION_NUMBER="dev"
+CONFIG_VERSION_REPO="https://downloads.creatordev.io/pistachio/marduk/packages"
```

Comment:
please use http instead of https.  SSL certificate is currently failing to validate on the downloads server